### DISCUSSION TITLE
What Kind of Crossing is this?

### DISCUSSION THREAD

**Discussion by kiz-tech** [streetcomplete/StreetComplete#5290] [Ideas]
Hy,

in the App there is the question "what Kind of Crossing is this" and one of the answer is "marked" or something like that (I have used the app in german) and in the picture I can see a zebra crossing.

The zebra Crossing is in Germany a defined mark, that the pedestrians can go and the cars has top stop. In other cases of marking, the pedestrians have to stop in my opinion and wait for the cars.

What I should answer in the case of other marking? Yes (because there is a marking) or no (because there is no zebra crossing)? Mybe the App could ask the Kind of marking also in this question? Or add into the answer "(all kinds of marking)" and add another question to define the marking?

Best Regards, Daniel

**Comment by westnordost** [comment]
What kind of other marking do you mean?

**Comment by kiz-tech** [comment]
Here some pictures:
![IMG_20231004_160910](https://github.com/streetcomplete/StreetComplete/assets/146853376/8853d9d0-b612-43b2-8c15-c7d53dea315d)
![IMG_20231004_161352](https://github.com/streetcomplete/StreetComplete/assets/146853376/0920ad50-5b3c-4ac7-abe0-0c0a7be068c0)
![IMG_20231004_161513](https://github.com/streetcomplete/StreetComplete/assets/146853376/828826dd-1d3f-449b-9880-2a203dbba223)

**Reply by mnalis** [reply]
**Yes, it is marked, so you should answer `marked`.** 

Specific type of marking (e.g. _"is it zebra or some other markings"_ doesn't matter). If you have suggestion how the wording might be improved to remove doubt, please suggest (while the issue is still fresh in your mind). But I'm not sure if it can be improved without using much more screen space (of which there is very little!)

In essence, when answering `marked` StreetComplete will just tag that node with [crossing=marked](https://wiki.openstreetmap.org/wiki/Tag:crossing=marked), which is explained on the wiki as:

> This tag describes a pedestrian crossing that is marked but not controlled by traffic lights or people. 

---

For more advanced users who care deeply about specific types of markings and other details of pedestrian crossings, [SCEE](https://wiki.openstreetmap.org/wiki/SCEE) fork of StreetComplete offers multiple answers there, which add multiple detailed tags e.g.:

![small_Screenshot_20231008_163855_de westnordost streetcomplete expert](https://github.com/streetcomplete/StreetComplete/assets/156656/dc7f1848-6aac-4c6a-9e0c-4bc9ac3199c1) ![small_Screenshot_20231008_170835_de westnordost streetcomplete expert](https://github.com/streetcomplete/StreetComplete/assets/156656/cb5209b4-e9cc-407b-9bdc-0ce706504102)

But for majority of regular users, the standard StreetComplete just asks about the most important part about type the crossing:

- best/safest is existence of **pedestrian traffic lights** (without regard to specific type, or even existence, of any road markings)
- some **road markings** (regardless of their type), but without any traffic lights, is in the middle (better than nothing)
- worst/least safe is **nothing** - not any road markings, nor any traffic lights

**Comment by kiz-tech** [comment]
> Yes, it is marked, so you should answer `marked`.

OK. Thank you. Can the information "(all kinds of marking)" (in all languages) added to the answer to avoid wrong answers?

**Reply by mnalis** [reply]
Possibly. It is kind of long even in English, but mockup still looks passable.
Other languages might be even longer, thus covering more of the picture though.

![Screenshot_20231010_025924_de westnordost streetcomplete_long](https://github.com/streetcomplete/StreetComplete/assets/156656/f402dee1-9483-4b42-bf85-f7d1d77ae344)

If accepted, `Marked (any kind of markings)` might be better English translation? (perhaps native English speakers will jump in)

**Reply by FloEdelmann** [reply]
What about just "Any kind of marking"?

**Reply by mnalis** [reply]
> What about just "Any kind of marking"?

@FloEdelmann Good by itself, but IMHO does not match the question _"What kind of crossing is this?"_. (while original text works as an answer: _"(this is) **marked** (kind of crossing)"_)

Perhaps if the question was also changed? (I can only come up with non-ideal _"How is this crossing regulated?"_, any better ideas?)

**Reply by matkoniecz** [reply]
`Marked (any surface road marking counts)` ?

**Reply by riQQ** [reply]
`With any (road) markings`. With this, it's more in line with `Without road markings`.
